Proton/Sources/EditorCommand/Commands/ListCommand.swift

```diff
@@ -107,8 +110,10 @@ public class ListCommand: EditorCommand {
             let paraStyle = value as? NSParagraphStyle
             let mutableStyle = ListTextProcessor().updatedParagraphStyle(paraStyle: paraStyle, listLineFormatting: editor.listLineFormatting, indentMode: .indent)
             editor.addAttribute(.paragraphStyle, value: mutableStyle ?? editor.paragraphStyle, at: range)
+            editor.typingAttributes[.paragraphStyle] = mutableStyle ?? editor.paragraphStyle
         }
         editor.addAttribute(.listItem, value: attrValue, at: selectedRange)
+        editor.typingAttributes[.listItem] = attrValue
```

Comment:
This fixes the issue mentioned in #59.
When adding a list option between two regular paragraphs, add the correct attributes to the range but the `typingAttributes` still contain the regular values and starting to type would then remove the indentation and bullet point.